docs/setup-and-configure/docker-install.md

```diff
@@ -69,3 +70,19 @@ Usage:
 ```bash
 docker run -it -v data:/cheqd --rm --entrypoint node-runner -e NODE_MONIKER=<moniker> -e GENESIS="<content>" cheqd-node
 ```
+
+### Mainnet launch
+
+For simple running node which can be connected to `cheqd-mainnet-1` the next steps can be useful:
+* Go to `docker/cheqd_node/mainnet`
+* Edit parameters `NODE_MONIKER` and `OUT_MOUNT_POINT` in the `.env` file and run:
+
+```
+docker-compose up
```

Comment:
I'd change this to explicitly specify the env file used by Compose. Otherwise if there are multiple files for env (this is one way, btw, we can ourselves have multiple env files for different types of deployments) https://docs.docker.com/engine/reference/commandline/service_create/#add-bind-mounts-volumes-or-memory-filesystems